sdk/tables/data-tables/src/TableTransaction.ts

```diff
@@ -279,15 +275,6 @@ export class InternalTableTransaction {
       this.resetableState.changesetId
     );
 
-    const options: ServiceClientOptions = this.clientOptions;
-
-    if (isTokenCredential(this.credential)) {
-      options.credentialScopes = STORAGE_SCOPE;
```

Comment:
how is this being done now?

Reply:
This is no longer needed in the Transaction since we are now using the same client that gets instantiated in `TableClient`. Before we were creating a new instance, but now is no longer needed